PKGBUILD

```diff
@@ -12,7 +12,7 @@ depends=('fftw' 'alsa-lib' 'ncurses' 'iniparser' 'sndio' 'portaudio')
 optdepends=('pulseaudio' 'pipewire')
 makedepends=('sndio' 'portaudio' 'libpulse' 'm4' 'automake' 'autoconf' 'libpipewire')
 source=("$pkgname-$pkgver.tar.gz::https://github.com/karlstav/cava/releases/download/0.10.3/${pkgname}-${pkgver}.tar.gz")
-sha512sums=('15fa550da266d3d4b917da0e6b5611bd7b82e5d30d15e3d728b0e3d5a92387787156e3e32f632f68b9c5e217a023bc2b3ca4ffd48ac6e03be1519ec2b504b13a')
+sha512sums=('35ee4821e094c2ea567012329c21a8283450100eada4f7e33f7decd935db94adbefd2a2a41801e0ef49eea0dcd859c397c5b3dc7316f0640394d67e3b658876c')
```

Comment:
Where did you get that hash from??    
![Screenshot_2024-12-15_15-09-29](https://github.com/user-attachments/assets/02dc008c-ed95-48df-8015-6696e503cebe)